### ISSUE TITLE
Log in Problem

### ISSUE THREAD

**Issue by Nayandey79** [kasmtech/workspaces-issues#135]
Hello,
I have apparently completed the installation, but I cannot log in with the login and password provided by the log, it says "error during request - please contact system administrator" (Ubuntu 20.4.3, 2vcpu, 4gb ram, 50gb disk on KVM VM on debian 11 server host).
Am I missing something?

**Comment by mmcclaskey** [comment]
Hello. Can you please provide the following:

From your browser, can you open up dev tools, then try to log in again, and then go to the console tab in dev tools and see if there was additional information about the error posted there. Additionally, SSH to the server and run the following command prior to logging in, provide the output that occurred when you attempted to login.

`sudo docker logs -f --tail 30 kasm_api`

**Comment by ondrejpales** [comment]
Happened to me also with this scenario:
- After installing 1.10.0.238225 on Ubuntu Server I was able to log in using the autogenerated admin credentials.
- I then created my own user with admin privileges, I loged in as this user, run some sessions. 
- Then I just closed the browser window (without properly loging out). 
- After that when I opened the browser (Firefox) again and accessed the web interface, it appeared as logged in, but all the values on the dashboard were zero or empty. 
- I tried clearing browser's local storage and cookies which got me to the Log in screen.
- But after that I was unable to log in with my newly created admin user. I tried the default admin user also, but I could not log in either. The only way was to completely uninstall and install again.

**Comment by Nayandey79** [comment]
Login at the console tab I was able to see what was the problem, disk was full and it could not generate user  , thing that I do not understand as only 24Gig where used of the 50Gig.
I did the disk again with 100gig and now it's working.

**Comment by ondrejpales** [comment]
> Login at the console tab I was able to see what was the problem, disk was full and it could not generate user , thing that I do not understand as only 24Gig where used of the 50Gig.

This might have been the problem in my case, too, as I recall seeing the 24G being used out of 50G disk also, which I was hoping to solve later, but wasn't able to due to this log in problem. I then stopped using kasm, so I couldn't troubleshoot anymore. I will try it with a larger disk, too.

**Comment by mmcclaskey** [comment]
@ondrejpales @Nayandey79 , sorry to hear you are having issues. I am curious to get more information as Kasm should only be taking about 20GB if you chose to install all default images. If you are able to duplicate the issue where your HD gets filled and you had a 50GB volume on a fresh VM, would you mind posting the output from a few commands?

sudo docker system df -v
df -h
sudo systemctl status docker | cat

This will verify how your HD is partitioned, where the docker data directory is. If the full 50GB is assigned to root and/or the docker data directory has access to that full 50GB and it is using all of it, the docker system df command will tell us what is using all the space.

**Comment by ondrejpales** [comment]
> If you are able to duplicate the issue where your HD gets filled and you had a 50GB volume on a fresh VM, ...

After creating a new VM and installing Kasm on Ubuntu again I can confirm that in my case nothing was wrong with Kasm and the whole scenario I posted earlier was a pure coincidence. The real problem was caused by Ubuntu installer which in default settings using LVM creates only one volume spanning mere 50% of the disk (as documented here https://bugs.launchpad.net/ubuntu/+source/subiquity/+bug/1907128). I did not notice this and thought I had the whole 50GB disk available, but in reality the volume was only 25GB big, which quickly filled up.

I'm now running Kasm on a new VM with the whole 50GB volume and so far there are no problems.
